backend/tournesol/tests/test_api_comparison.py

```diff
@@ -57,9 +57,9 @@ def setUp(self):
 
         At least 4 videos and 2 users with 2 comparisons each are required.
         """
-        user = User.objects.create(username=self._user)
-        user2 = User.objects.create(username=self._user2)
-        other = User.objects.create(username=self._other)
+        user = User.objects.create(username=self._user, email="user@test")
+        user2 = User.objects.create(username=self._user2, email="user2@test")
+        other = User.objects.create(username=self._other, email="other@test")
```

Comment:
With the adaptation of these fields in our tests, we should maybe consider using factories ?
This issue could be handled in another PR if you want.

Reply:
Hi @sandre35 

It's a good idea, it will make the tests development easier. It's indeed a feature that would be great in another pull request, as it will contains a refactor of several tests not related to the current PR.

Do you have any experience with test fixtures? I know there are several way to implement them:
- using the built-in fixtures provided by Django
- using a third-party module like [factory_boy](https://factoryboy.readthedocs.io/en/stable/index.html)

I only have used Django fixtures in my previous projects, and I liked the possibility to create and edit the fixtures directly in the Django admin interface, and dump them in JSON files afterward. I never used `factory_boy` or any similar tool (`Faker`?), but they seem good option to consider.

Reply:
I used "factory_boy" in a previous project, its integration with Django models to define factories is very handy:
https://factoryboy.readthedocs.io/en/stable/orms.html#django

Such factories are especially useful to focus on the fields that are relevant to the current test. So they are probably easier to maintain, compared to raw fixtures.

Reply:
Because there seem to be an unanimous aggreement, I made the following ticket: https://github.com/tournesol-app/tournesol/issues/396. Any of you can feel free to take it.